self-hosted/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8.16-slim-bullseye
+FROM python:3.8.18-slim-bullseye
```

Comment:
Any interest in bumping the Debian version on this as well, to `-slim-bookworm` or would you rather we handle that in a separate PR?

Reply:
that's the plan -- I need to do this bump first because 16 is not available on bookworm